README.md

```diff
@@ -1,2 +1,108 @@
-# doppler-html-editor-api
-API Rest from Doppler HTML Editor
+# Doppler HTML Editor API
+
+## Campaigns
+`GET /accounts/{accountName}/campaigns/{campaignId}/content/design`
```

Comment:
I would use code blocks because it will allow to add more lines with the header or the request:

```suggestion
\```
GET /accounts/{accountName}/campaigns/{campaignId}/content/design
\```
```

Reply:
Great! Fixed 👍